### PR TITLE
fixup! ASoC: Intel: soc-acpi-intel-lnl-match: add rt712_vb + rt1320 support

### PR DESCRIPTION
**Pull request by bardliao**
We need to use snd_soc_acpi_intel_sdca_is_device_rt712_vb to check if the rt712 is vb.

### CODE REVIEW

File: sound/soc/intel/common/soc-acpi-intel-lnl-match.c [hunk 1]
```diff
@@ -546,6 +547,7 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_lnl_sdw_machines[] = {
 		.link_mask = BIT(1) | BIT(2),
 		.links = lnl_sdw_rt712_vb_l2_rt1320_l1,
 		.drv_name = "sof_sdw",
+		.machine_check = snd_soc_acpi_intel_sdca_is_device_rt712_vb,
```
Comment (ujfalusi) [inline sound/soc/intel/common/soc-acpi-intel-lnl-match.c:550]: how did you missed this in #5223? ;)

**Review by ujfalusi — DISMISSED** [review]

File: sound/soc/intel/common/soc-acpi-intel-lnl-match.c (hunk 1 above)
Comment (bardliao) [reply sound/soc/intel/common/soc-acpi-intel-lnl-match.c:550]: > how did you missed this in #5223? ;)

There is no proper acpi table in my RVP. And I just realized that why don't I need to remove the .machine_check in my test.

Comment (bardliao) [reply sound/soc/intel/common/soc-acpi-intel-lnl-match.c:550]: Also, the rt1320 ADR was wrong. I got a new AIOC from Realtek with the new rt1320 version today.

Comment (ujfalusi) [reply sound/soc/intel/common/soc-acpi-intel-lnl-match.c:550]: Right, you know it best...

**Review by ujfalusi — APPROVED** [review]

**Review by ranj063 — APPROVED** [review]

**Pull request merged**